specification.rst

```diff
@@ -3150,15 +3138,27 @@ editor
 editorial-director
     Managing editor ("Directeur de la Publication" in French)
 
+editor-translator
+    Automatically constructed if editor and translator are identical
```

Comment:
I don't think this _has_ to be automatically constructed

```suggestion
    Editor and translator of a work. Can be automatically generated if editor and translator variables are identical
```

Reply:
Question is if this will ever be manually entered by users?

Reply:
But I wouldn't mind your suggestion.

Reply:
Depends on the tool -- for people using CSL-YAML I could see that, also I think it's a variable in Biblatex (?) so in automated mapping from other formats. 
Maybe we should use `should` instead of `can` though if we want processors to behave consistently.

Reply:
Biblatex, no, I don't think so.
But I see your point with CSL-YAML. That could simplify things a bit.
Re `should`: I think it `must` be always automatically constructed if editor and translator are identical. That does not prevent users from just using the `editor-translator` variable directly in CSL-YAML.

Reply:
But I think we understand `should` in a rather prescriptive sense. So yeah, `should` is appropriate.

Reply:
No, we follow very specific conventions for Must, should, etc as listed at the top of the specs under Terminology:
https://tools.ietf.org/html/rfc2119 
So if this should always happen, `must` is the right term. I've changed it above.

Reply:
Done